docs/http-api/images.rst

```diff
@@ -0,0 +1,164 @@
+Images
+======
+
+Get list of images
+------------------
+
+Request
+^^^^^^^
+
+.. code-block::
+
+  GET /images?model_id=1
+
+* `model_id`: id of model to get images for
+
+Response
+^^^^^^^^
+
+* `200`:
+
+.. code-block:: json
+
+  [
+    {
+      "name": "task_one/image_name:latest",
+      "id": 1,
+      "model_id": 1,
+      "author": "user_name",
+      "creation_date": "1970-01-01 00:00:00.000000 ",
+      "params": {
+        "type": "ebonite.build.docker.DockerImage",
+        "name": "image_name",
+        "tag": "latest",
+        "repository": "task_one",
+        "registry": {"type": "ebonite.build.docker.DefaultDockerRegistry"}
+      }
+    }
+  ]
+
+* `404`: if given model doesn't exist
+
+
+Get image
+---------
+
+Request
+^^^^^^^
+
+.. code-block::
+
+  GET /image/<:id>
+
+* `id`: id of image to get
+
+Response
+^^^^^^^^
+
+* `200`:
+
+.. code-block:: json
+
+  {
+    "name": "task_one/image_name:latest",
+    "id": 1,
+    "model_id": 1,
+    "author": "user_name",
+    "creation_date": "1970-01-01 00:00:00.000000 ",
+    "params": {
+      "type": "ebonite.build.docker.DockerImage",
+      "name": "image_name",
+      "tag": "latest",
+      "repository": "task_one",
+      "registry": {"type": "ebonite.build.docker.DefaultDockerRegistry"}
+    }
+  }
+
+* `404`: if given image doesn't exist
+
+
+Build image
+-----------
+
+Request
+^^^^^^^
+
+.. code-block::
+
+  POST /images
```

Comment:
crud and actions on resources must be separated, i propose smt like '/images/build'

Reply:
or hat title was misleading and it is actually not an "build action", but just a c operation from crud

Reply:
How could crud and actions be separated if `Ebonite` class encapsulates them to work all together consistently? Moreover, what's the motivation to separate them in the first place?

Reply:
NB: yes, this operation is both create and build

Reply:
ok